theseus/core/robust_loss.py

```diff
@@ -11,13 +11,14 @@
 
 
 class RobustLoss(abc.ABC):
+    # .suqre() make it easier to compare with residual.square(), which is x
     @classmethod
     def evaluate(cls, x: torch.Tensor, log_radius: torch.Tensor) -> torch.Tensor:
-        return cls._evaluate_impl(x, log_radius.exp())
-
+        return cls._evaluate_impl(x, log_radius.square()) 
```

Comment:
We should keep this as `exp()`.

Reply:
Sure. Could I know the initiative behind it?

Reply:
Since square() is not monotonous, I personally prefer 'exp()', which is more useful when we try to learn a robust loss radius.

Reply:
> 

Oh! That's great! Because at a very early time, I tried to let the system automatically learn the Radius, and I failed at that time (performance was not good). But with the new version, the training outcome may change.